pkg/minikube/cruntime/docker.go

```diff
@@ -252,3 +265,34 @@ func (r *Docker) ContainerLogCmd(id string, len int, follow bool) string {
 func (r *Docker) SystemLogCmd(len int) string {
 	return fmt.Sprintf("sudo journalctl -u docker -n %d", len)
 }
+
+// Preload preloads docker with k8s images:
+// 1. Copy over the preloaded tarball into the VM
+// 2. Extract the preloaded tarball to the correct directory
+// 3. Remove the tarball within the VM
+func (r *Docker) Preload(k8sVersion string) error {
+	tarballPath := download.TarballPath(k8sVersion)
+	dest := "/tmp/preloaded.tar"
```

Comment:
.tar might be confusing - how about /tmp/preloaded.tar.lz4 ?

Reply:
done